vendor/bat-native-ledger/src/bat/ledger/internal/bat_util.cc

```diff
@@ -29,4 +29,15 @@ std::string ConvertToProbi(const std::string& amount) {
   return before_dot + after_dot + rest_probi;
 }
 
+double ProbiToDouble(const std::string& probi) {
+  const size_t size = probi.size();
+  std::string amount = "0";
+  if (size > 18) {
+    amount = probi;
```

Comment:
const int32_t probi_size = 18;

(as per database_contribution.cc) - perhaps this should be a more accessible const somewhere?

Reply:
it's temporary thing while we are migrating to double. Probi will go away as soon as possible